### PR TITLE
net-im/skypeforlinux: Update descriptions to reflect changes away fro…

### PR DESCRIPTION
**Pull request by shentino**
…m P2P architecture

Closes: https://bugs.gentoo.org/621684

### CODE REVIEW

File: net-im/skypeforlinux/skypeforlinux-5.4.0.1.ebuild [hunk 1]
```diff
@@ -7,7 +7,7 @@ MULTILIB_COMPAT=( abi_x86_64 )
 
 inherit pax-utils rpm multilib-build
 
-DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
+DESCRIPTION="Internet Telephony (VoIP) and messaging client, with support for audio and video"
```
Comment (jonasstein) [inline net-im/skypeforlinux/skypeforlinux-5.4.0.1.ebuild]: could you make the capitalization consistent, please? Does "Telephony" not include audio? 
I would suggest to find a better, clearer DESC. (note the 80 char limit)

**Comment by karolgrudzinski** [comment]
I propose again using description similar to net-libs/tox's:
> VoIP, instant messaging, and audio/video calling client
It describes feature list without using marketing b***s**t and is much shorter (56 characters).

**Comment by shentino** [comment]
At this point it's migrated beyond telephony.

I'm honestly inclined to consider it as just another instant messaging app with support for audio and video.  After the merge with msn messenger and especially its integration with Xbox live it has evolved from the P2P telephony app it once was.

**Comment by karolgrudzinski** [comment]
@shentino Skype stopped using P2P after it was acquired by Microsoft.

**Comment by shentino** [comment]
@karolgrudzinski I completely agree.

Removing the reference to P2P is only part of the change I'm wanting to make with the description.  The way things are going it's just Yet Another IM client at this point.

**Comment by karolgrudzinski** [comment]
Currently Skype is Just-Yet-Another-Not-So-Great IM client so I think that maybe we put too much effort and time to make package description perfect :-). Just make it simple and don't overthink it.

**Comment by shentino** [comment]
That's the plan :)

Yes, it's a bikeshed issue, but I think there's still a point to making sure it's good enough people won't want to repaint it later.

**Comment by titanofold** [comment]
Closed by commit 1fad828b20b408f63cd6ceb6f8bf681766c3d319

**Pull request closed without merging**